src/less/carousel/base/carousel.less

```diff
@@ -197,6 +196,15 @@
     }
 }
 
+div.carousel {
```

Comment:
Is there any reason for the increased specificity here? (`div` & `span`)?

Reply:
Yes. I only want to leverage margin-collapse for block level div, not span.

Reply:
We allow for inline `span` carousels?

Reply:
... I should add a comment to the code.

Reply:
I don't think we have a real-life use case for inline span carousels, but I'm putting it in here to cover all bases.

Reply:
Note: I am not adding an example of inline span carousels to the website. It will remain on the secret menu ;-)